### PR TITLE
AutoCorrect files/zh-cn/web/api/f*/

### PR DESCRIPTION
**Pull request by huacnlee**

### CODE REVIEW

**Comment by yin1999** [comment]
@huacnlee, conflict, please rebase

**Comment by huacnlee** [comment]
@yin1999 Just updated

File: files/zh-cn/web/api/fetch_api/using_fetch/index.md [hunk 1]
```diff
@@ -368,9 +368,9 @@ fetch(myRequest)
 
 你会用到的最常见的 response 属性有：
 
-- {{domxref("Response.status")}} — 整数（默认值为 200）为response的状态码。
-- {{domxref("Response.statusText")}} — 字符串（默认值为 ""），该值与 HTTP 状态码消息对应。 注意：HTTP/2 [不支持](https://fetch.spec.whatwg.org/#concept-response-status-message)状态消息
-- {{domxref("Response.ok")}} — 如上所示，该属性是来检查 response 的状态是否在 200 - 299（包括200 和 299）这个范围内。该属性返回一个布尔值。
+- {{domxref("Response.status")}} — 整数（默认值为 200）为 response 的状态码。
+- {{domxref("Response.statusText")}} — 字符串（默认值为 ""），该值与 HTTP 状态码消息对应。注意：HTTP/2 [不支持](https://fetch.spec.whatwg.org/#concept-response-status-message) 状态消息
```
Comment (yin1999) [inline files/zh-cn/web/api/fetch_api/using_fetch/index.md:372]:
```suggestion
- {{domxref("Response.statusText")}} — 字符串（默认值为 ""），该值与 HTTP 状态码消息对应。注意：HTTP/2 [不支持](https://fetch.spec.whatwg.org/#concept-response-status-message)状态消息
```

File: files/zh-cn/web/api/fetchevent/respondwith/index.html [hunk 2]
```diff
@@ -5,9 +5,9 @@
...
+<p>{{domxref("FetchEvent")}} 接口的 <strong><code>respondWith()</code></strong> 方法旨在包裹代码，这些代码为来自受控页面的 request 生成自定义的 response。这些代码通过返回一个 {{domxref("Response")}} 、 <a class="external external-icon" href="http://fetch.spec.whatwg.org/#concept-network-error">network error</a> 或者 <code><a class="external external-icon" href="http://fetch.spec.whatwg.org/#concept-fetch">Fetch</a>的方式</code>resolve。</p>
 
-<p>有关跨域内容污染的渲染端安全检测与 {{domxref("Response")}} 体的透明度（或者不透明度）相关联，而不是URL。如果request是一个顶级的导航，而返回值是一个类型属性不透明的 {{domxref("Response")}}（即不透明响应体），一个 <a class="external external-icon" href="http://fetch.spec.whatwg.org/#concept-network-error">network error</a> 将被返回给 <code><a class="external external-icon" href="http://fetch.spec.whatwg.org/#concept-fetch">Fetch</a>。所有成功（非网络错误）响</code>应的最终URL是请求的URL。</p>
+<p>有关跨域内容污染的渲染端安全检测与 {{domxref("Response")}} 体的透明度（或者不透明度）相关联，而不是 URL。如果 request 是一个顶级的导航，而返回值是一个类型属性不透明的 {{domxref("Response")}}（即不透明响应体），一个 <a class="external external-icon" href="http://fetch.spec.whatwg.org/#concept-network-error">network error</a> 将被返回给 <code><a class="external external-icon" href="http://fetch.spec.whatwg.org/#concept-fetch">Fetch</a>。所有成功（非网络错误）响</code>应的最终 URL 是请求的 URL。</p>
```
Comment (yin1999) [inline files/zh-cn/web/api/fetchevent/respondwith/index.html:10]:
```suggestion
<p>有关跨域内容污染的渲染端安全检测与 {{domxref("Response")}} 体的透明度（或者不透明度）相关联，而不是 URL。如果 request 是一个顶级的导航，而返回值是一个类型属性不透明的 {{domxref("Response")}}（即不透明响应体），一个 <a href="https://fetch.spec.whatwg.org/#concept-network-error">network error</a> 将被返回给 <a href="https://fetch.spec.whatwg.org/#concept-fetch"><code>Fetch</code></a>。所有成功（非网络错误）响应的最终 URL 是请求的 URL。</p>
```

File: files/zh-cn/web/api/file_api/using_files_from_web_applications/index.html [hunk 3]
```diff
@@ -252,31 +252,31 @@ <h2 id="例子：显示用户选择的图片的缩略图">例子：显示用户
...
 </pre>
 
-<p>这个对象URL是一个标识{{ domxref("File") }}对象的字符串。每次你调用{{ domxref("window.URL.createObjectURL()") }}，就会产生一个唯一的对象URL，即使是你对一个已创建了对象URL的文件再次创建一个对象URL。每个创建了的对象URL必须要释放。当文档关闭时，它们会自动被释放。如果你的网页要动态使用它们，你需要显式调用 {{ domxref("window.URL.revokeObjectURL()") }}来释放它们：</p>
+<p>这个对象 URL 是一个标识{{ domxref("File") }}对象的字符串。每次你调用{{ domxref("window.URL.createObjectURL()") }}，就会产生一个唯一的对象 URL，即使是你对一个已创建了对象 URL 的文件再次创建一个对象 URL。每个创建了的对象 URL 必须要释放。当文档关闭时，它们会自动被释放。如果你的网页要动态使用它们，你需要显式调用 {{ domxref("window.URL.revokeObjectURL()") }}来释放它们：</p>
 
 <pre class="brush: js notranslate">window.URL.revokeObjectURL(objectURL);
 </pre>
 
-<h2 id="示例：使用对象URL来显示图片">示例：使用对象URL来显示图片</h2>
+<h2 id="示例：使用对象URL来显示图片">示例：使用对象 URL 来显示图片</h2>
```
Comment (yin1999) [inline files/zh-cn/web/api/file_api/using_files_from_web_applications/index.html:275]:
```suggestion
<h2 id="示例：使用对象_url_来显示图片">示例：使用对象 URL 来显示图片</h2>
```

**Review by yin1999 — CHANGES_REQUESTED** [review]

File: files/zh-cn/web/api/fetch_api/using_fetch/index.md [hunk 4]
```diff
@@ -21,8 +21,8 @@ translation_of: Web/API/Fetch_API/Using_Fetch
 
 请注意，`fetch` 规范与 `jQuery.ajax()` 主要有以下的不同：
 
-- 当接收到一个代表错误的 HTTP 状态码时，从 `fetch()` 返回的 Promise **不会被标记为 reject**，即使响应的 HTTP 状态码是 404 或 500。相反，它会将 Promise 状态标记为 resolve （如果响应的 HTTP 状态码不在 200 - 299 的范围内，则设置 resolve 返回值的 {{domxref("Response/ok", "ok")}} 属性为 false ），仅当网络故障时或请求被阻止时，才会标记为 reject。
-- `fetch` **不会发送跨域 cookies**，除非你使用了 _credentials_ 的[初始化选项](/zh-CN/docs/Web/API/fetch#参数)。（自[2018 年 8 月](https://github.com/whatwg/fetch/pull/585)以后，默认的 credentials 政策变更为 `same-origin`。Firefox 也在 61.0b13 版本中进行了修改）
+- 当接收到一个代表错误的 HTTP 状态码时，从 `fetch()` 返回的 Promise **不会被标记为 reject**，即使响应的 HTTP 状态码是 404 或 500。相反，它会将 Promise 状态标记为 resolve（如果响应的 HTTP 状态码不在 200 - 299 的范围内，则设置 resolve 返回值的 {{domxref("Response/ok", "ok")}} 属性为 false），仅当网络故障时或请求被阻止时，才会标记为 reject。
+- `fetch` **不会发送跨域 cookie**，除非你使用了 _credentials_ 的[初始化选项](/zh-CN/docs/Web/API/fetch#参数)。（自 [2018 年 8 月](https://github.com/whatwg/fetch/pull/585) 以后，默认的 credentials 政策变更为 `same-origin`。Firefox 也在 61.0b13 版本中进行了修改）
```
Comment (yin1999) [inline files/zh-cn/web/api/fetch_api/using_fetch/index.md:25]:
```suggestion
- `fetch` **不会发送跨域 cookie**，除非你使用了 _credentials_ 的[初始化选项](/zh-CN/docs/Web/API/fetch#参数)。（自 [2018 年 8 月](https://github.com/whatwg/fetch/pull/585)以后，默认的 credentials 政策变更为 `same-origin`。Firefox 也在 61.0b13 版本中进行了修改）
```

File: files/zh-cn/web/api/fetch_api/using_fetch/index.md [hunk 5]
```diff
@@ -34,7 +34,7 @@ fetch('http://example.com/movies.json')
 
 这里我们通过网络获取一个 JSON 文件并将其打印到控制台。最简单的用法是只提供一个参数用来指明想 `fetch()` 到的资源路径，然后返回一个包含响应结果的 promise（一个 {{domxref("Response")}} 对象）。
 
-当然它只是一个 HTTP 响应，而不是真的 JSON。为了获取JSON的内容，我们需要使用 {{domxref("Response.json()", "json()")}} 方法（该方法返回一个将响应 body 解析成 JSON 的 promise）。
+当然它只是一个 HTTP 响应，而不是真的 JSON。为了获取 JSON 的内容，我们需要使用 {{domxref("Response.json()", "json()")}} 方法（该方法返回一个将响应 body 解析成 JSON 的 promise）。
 
 > **备注：** [Body](#Body) 还有其他相似的方法，用于获取其他类型的内容。
```
Comment (yin1999) [inline files/zh-cn/web/api/fetch_api/using_fetch/index.md:39]:
```suggestion
> **备注：** [Body](#body) 还有其他相似的方法，用于获取其他类型的内容。
```

**Pull request merged**